nselib/data/http-fingerprints.lua

```diff
@@ -7044,7 +7044,90 @@ table.insert(fingerprints, {
 ----        Open Source CMS checks          ----
 ------------------------------------------------
 
--- Broad wordpress version identification
+-- Wordpress versions (Scraping readme file)
+table.insert(fingerprints, {
+  category = 'cms',
+  probes = {
+    {
+      path = '/readme.html',
+      method = 'GET'
+    }
+  },
+  matches = {
+    {
+      match = '[V|v]ersion ([0-9 .]*)',
+      output = 'Wordpress version: \\1'
+    }
+  }
+});
+
+-- Wordpress versions (Scraping metatags and wp-includes)
+table.insert(fingerprints, {
+  category = 'cms',
+  probes = {
+    {
+      path = '/',
+      method = 'GET'
+    }
+  },
+  matches = {
+    {
+      match = '<meta name="generator" content="WordPress ([0-9 .]*)" />',
+      output = 'WordPress version: \\1'
+    },
+    {
+      match = '/wp-includes\\/js\\/wp-emoji-release.min.js?ver=([0-9 .]*)',
```

Comment:
Lots of urls like this could work besides wp-emoji-release. Can we change this to `'/wp-includes/js/[%w.-]+.js?ver=([0-9.]+)'`? Also note that "/" does not need to be escaped in lua patterns.